### ISSUE TITLE
Feature request: Add draft-04 schema support

### ISSUE THREAD

**Issue by kirpichyov** [GrantBirki/json-yaml-validate#68]
Hi! I am trying to use action to verify the API doc file which follows the OpenAPI 3.0.1 schema ([Schema reference](https://github.com/OAI/OpenAPI-Specification/blob/main/schemas/v3.0/schema.json)). 

```yaml
      - name: Verify api-doc.json
        uses: GrantBirki/json-yaml-validate@v3.0.0
        with:
          mode: "fail"
          comment: "true"
          files: "api-doc.json"
          json_schema: "./schemas/open-api-3_0.json"
          json_schema_version: "draft-04"
```

But it gives an error:

```
Warning: json_schema_version 'draft-04' is not supported. Defaulting to 'draft-07'

/home/runner/work/_actions/GrantBirki/json-yaml-validate/v3.0.0/node_modules/ajv/dist/core.js:147
                throw new Error(`no schema with key or ref "${schemaKeyRef}"`);
```

So it looks like OpenAPI 3.0.1 use draft-04 json schema which is supported by Ajv according to the [documentation](https://ajv.js.org/json-schema.html#draft-04) but not supported by the `json-yaml-validate`. I have tried to use other schemas in the `json_schema_version` like `draft-07` but it doesn't work. It would be great and very appreciable to add support for `draft-04` for the `json_schema_version` parameter.

**Comment by GrantBirki** [comment]
This does appear to be possible and something I can work on in the future: https://github.com/ajv-validator/ajv-draft-04

**Comment by GrantBirki** [comment]
@kirpichyov I have published a pre-release -> [`v3.2.0`](https://github.com/GrantBirki/json-yaml-validate/releases/tag/v3.2.0). Would you be able to try this out and see if it works for your use case and let me know how it goes? Thanks! 🙇

**Comment by kirpichyov** [comment]
It works now, thanks!
